borrowing_app/serializers.py

```diff
@@ -8,16 +9,34 @@
 class BorrowingSerializer(serializers.ModelSerializer):
     class Meta:
         model = Borrowing
-        fields = [
+        fields = ("expected_return_date", "book")
+
+    @staticmethod
+    def validate_book(value):
+        if value.inventory <= 0:
+            raise serializers.ValidationError("You can't borrow a book now")
+        return value
```

Comment:
Did you try to reproduce the error raise? does it come out of serializer first or model?

Reply:
I put screen how it's look like, It validate on serializer level